ui/src/lib/stores.ts

```
@@ -0,0 +1,120 @@
+
+interface DevicesData {
+    devices: any[];
+}
+
+interface WebSocketCommand {
+    command: string;
+    payload?: string;
+}
+
+// Type for the start functions used in readable stores
+type StartFunction<T> = (set: (value: T) => void) => (() => void) | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)